src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -361,6 +361,72 @@ def _find_or_create_unresolved_symbol(location, name, scope_limit=None,
     raise SymbolError(f"No Symbol found for name '{name}'.")
 
 
+def _find_or_create_psyclone_internal_cmp(node):
+    '''
+    Utility routine to return a symbol of the generic psyclone comparison
+    interface. If the interface does not exist in the scope it first adds
+    the necessary code to the parent module.
+
+    :returns: the comparison interface symbol.
+    :rtype: :py:class:`psyclone.psyir.symbols.Symbol`
+
+    :raises NotImplementedError: if there is no ancestor module container
+        on which to add the interface code into.
+    '''
+    try:
+        return node.scope.symbol_table.lookup_with_tag("psyclone_internal_cmp")
+    except KeyError:
+        container = node.ancestor(Container)
+        if container and not isinstance(container, FileContainer):
+            from psyclone.psyir.frontend.fortran import FortranReader
```

Comment:
Please disable the pylint warning for this import.

Reply:
Done